src/blocks/mod.rs

```diff
@@ -404,6 +409,23 @@ impl Block {
     }
 }
 
+impl Display for Block {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let work_render = if self.work.is_none() {
```

Comment:
I prefer this for various reasons, especially because there is no need to do an unwrap (even though it's safe, it could not be in the future if edited incorrectly by someone else):
```rust
match &self.work {
    Some(work_render) => work_render.to_string(),
    None => "No work".to_string(),
}
```